File: JAVA/introduction/basic program/JAVA2023/JAVA BY PW/factorial.java
import java.util.*;
public class factorial {
    public static void main(String[] arge){
        Scanner read = new Scanner(System.in);
        System.out.println("ente the number:");

    int n = read.nextInt();
    int fact=1;
    System.out.println("factorial number is:");

    for (int i=1; i<=n; i++) {
        fact*=i;
        System.out.println(fact);

    }
    //System.out.println(fact);
    }
}

File: JAVA/introduction/basic program/JAVA2023/factorial.c
#include<stdio.h>

    int main(){
    int n,i,p=1;
    printf("enter the number:\n");
    scanf("%d",&n);
    for ( i = n; i > 0; i--)
    {
        p=p*i;
    }
    printf("factorial is %d",p);
return 0;
}